en/docs/develop-components/deploy-a-containerized-application.md

```diff
@@ -0,0 +1,119 @@
+# Deploy a Containerized Application 
+
+Using Choreo, you can easily deploy applications written in different language frameworks (such as Java, Go, NodeJs, Python, etc.) on shared or private data planes using containers.
+
+Choreo supports deploying containerized applications for the following component types:
+
+- Service
+- Scheduled Trigger
+- Manual Trigger
+- REST API
+- Webhook
+
+## Connect your repository to Choreo
+
+To connect your GitHub repository to Choreo, you should authorize the [Choreo Apps](https://github.com/marketplace/choreo-apps) GitHub application to access your account or organization. When you attempt to connect your GitHub repository via the Component creation page, the Choreo Apps authorization prompt will appear.
+
+* Connect GitHub Repository
+    
+    ![Connect GitHub Repository](../assets/img/develop-components/deploy/create-component-authz-github-page.png){.cInlineImage-threeQuarter}
+
+* Authorize GitHub Application
+    
+    ![Authorize GitHub Application](../assets/img/develop-components/deploy/authz-choreo-github-app.png){.cInlineImage-half}
+
+* Grant Repository Access
+
+    ![Grant Repository Access](../assets/img/develop-components/deploy/github-repo-access.png){.cInlineImage-full}
+
+
+    The **Choreo Apps** requires the following permission:
+    
+    - Read access to issues and metadata
+    - Read and write access to code, pull requests, and repository hooks
+
+!!! note
+    You can [revoke access](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/reviewing-your-authorized-integrations#reviewing-your-authorized-github-apps) if you do not want Choreo to have access to your GitHub account. Choreo needs write access only to send pull requests to a user repository. Choreo does not directly push any changes to a repository.
+
+After granting access to the required repositories, you can choose a repository and an associated branch to connect to the Choreo component.
+For the containerized application deployments, you should select the **Dockerfile** as the **Build Preset** and provide **Dockerfile Path** and [**Docker Context Path**](https://docs.docker.com/build/building/context/#path-context) for the docker build.
+
+![Connected Dockerfile Repository](../assets/img/develop-components/deploy/create-component-connected-repo.png){.cInlineImage-full}
+
+The following table describes the individual fields in the **Connect Repository** form.
```

Comment:
Any reason to go with the term "form"? We have used the term "pane" based on the Google style guide.

Reply:
done